src/apirequest.ts

```diff
@@ -188,6 +188,18 @@ async function createAPIRequestAsync<T>(parameters: APIRequestParams) {
     parameters.mediaUrl = urlTemplate.parse(parameters.mediaUrl).expand(params);
   }
 
+  // Rewrite url if rootUrl is globally set
+  if (
+    parameters.context._options.rootUrl !== undefined &&
+    options.url !== undefined
+  ) {
+    const path = options.url.slice(parameters.context._options.rootUrl.length);
+    options.url = (parameters.context._options.rootUrl + '/' + path).replace(
```

Comment:
can we use `url.resolve` here too, like in the test?

Reply:
👍